### DISCUSSION TITLE
murena.io emails not usable with github

### DISCUSSION THREAD

**Discussion by denmedius** [community/community#73135] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello community.

My daughter, who enjoys a [murena.io](https://murena.io/) account, cannot use her @murena.io email with github : his github account has been immediately blocked with the following message :

> Our abuse detecting systems placed restrictions on the account because of the email address on the account.
> Before we can remove > these restrictions, we would need:
> * a non-disposable email address on the account
> * any throwaway emails (including forwarders) to be removed
> You can add an email address by following the steps here:
> * Adding an email address to your GitHub account
> You can then follow these steps to verify it:
> * Verifying your email address
> Once more, we'll need you to remove the current email address from your account.
> To clarify, we don't need anything 'traceable' to you, feel free to use protonmail or tutanota etc.
> (just examples, we don't have any particular recommendation here) it just can't be a "throwaway"
> or temporary domain for security and deliverability reasons.
> You are also welcome to connect to GitHub using a VPN or TOR node if and as you wish.

What is the official and precise reason why this murena.io domain is banned by github ?

Note: I've look at https://github.com/disposable/disposable-email-domains, and could not find this domain ... 

Thanks in advance.

**Comment by ghostinhershell** [comment]
Hi @denmedius, thanks so much for reaching out in the Community

The murena domain is currently not allowed as it has been used several times by bad actors for spammy or inauthentic activities. We will be able to review the status of your account if you change your email address.

To clarify, we don't need anything 'traceable' to you or a Google/Microsoft email address, feel free to use protonmail or tutanota etc. (just examples, we don't have any particular recommendation here). This is for security and deliverability reasons. You are also welcome to connect to GitHub using a VPN or TOR node as you wish.

**Reply by denmedius** [reply]
Thanks @ghostinhershell !

**Reply by ghostinhershell** [reply]
No problem! If my response does answer your question, can you mark my response as the answer here so that we can close the loop on this post?

**Comment by DominikB1** [comment]
git remote set-url origin NEW_URL

**Comment by DominikB1** [comment]
dominikbartos974@gmail.com
